pages/linux/fdisk.md

```diff
@@ -10,3 +10,27 @@
 - Start the partition manipulator:
 
 `fdisk {{/dev/sdX}}`
+
+- Once partitioning a disk, create a partition:
+
+`n`
+
+- Once partitioning a disk, delete a specific partition:
```

Comment:
Do you have to select the partition beforehand or do you have to enter the partition in a prompt?

Reply:
I think it gives you a prompt to pick a partition to delete, however if there is only 1 then it is instantly deleted.

Reply:
How about
```suggestion
- Once partitioning a disk,open a prompt to ask for a partition to delete:
```
With the old description I'd be way to paranoid that it deletes a partition without we knowing which.

Reply:
```suggestion
- Once partitioning a disk, select a partition to delete:
```

Reply:
Sounds good 👍🏽